src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -63,17 +65,20 @@ public ResponseEntity deleteTicket(@PathVariable("id") String id){
     }
 
     @PostMapping(value="/ticket", consumes="application/json", produces="application/json")
-    public Ticket createTicket(@RequestBody Ticket ticket){
-
+    public ModelAndView createTicket(@RequestBody Ticket ticket){
+       ModelAndView modelAndView = new ModelAndView();
        Ticket newTicket = null;
 
         try {
             newTicket= ticketService.save(ticket);
+            modelAndView.setViewName("success");
         } catch (Exception e) {
-
+            e.printStackTrace();
+            modelAndView.setViewName("error");
```

Comment:
Added exception handling to the catch block to trace and report error.
https://dzone.com/articles/9-best-practices-to-handle-exceptions-in-java

Reply:
It's good to handle exceptions well.
Go one step further.  Use proper logging instead of printStackTrace().
- Loggins has the concept of priority (error, debug, warn, etc.), which printStackTrace doesn't have.
- No one will see printStackTrace(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with printStackTrace.